x-pack/test/alerting_api_integration/common/fixtures/plugins/actions_simulators/server/swimlane_simulation.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  RequestHandlerContext,
+  KibanaRequest,
+  KibanaResponseFactory,
+  IKibanaResponse,
+  IRouter,
+} from 'kibana/server';
+
+export function initPlugin(router: IRouter, path: string) {
+  router.post(
+    {
+      path,
+      options: {
+        authRequired: false,
+      },
+      validate: {},
+    },
+    // Swimlane simulator: create an action pointing here, and you can get
+    // different responses based on the message posted. See the README.md for
+    // more info.
+    async function (
+      context: RequestHandlerContext,
+      req: KibanaRequest<any, any, any, any>,
+      res: KibanaResponseFactory
+    ): Promise<IKibanaResponse<any>> {
+      const { body } = req;
+
+      return jsonResponse(res, 202, {
```

Comment:
It would be good to test error conditions as well.  

The slack simulator does some interesting things, based on the incoming input, and also makes some of that data available through an "extended" server API, so we can retrieve the data that is actually being sent to the service simulator.  Some of that was done to make sure the action variables got set correctly, which is more of a test of how the alerts and actions interact, than the actions themselves, so ... more elaborate than you probably need.